src/content/tutorials/user-auth-with-auth0/_index.md

```diff
@@ -670,6 +675,18 @@ $applicationName.$subdomain.workers.dev/auth
 
 Following this example, the callback URL for my application is `my-auth-test.signalnerve.workers.dev/auth`.
 
+#### Setting the salt
+
+In order to safely store user IDs (the `sub` value from Auth0), we should always refer to them by an encrypted value, which we can generate using the `crypto.subtle.digest` function in the Web Crypto API. In order to do this, we need to set a _salt_: a secret value that is included in the text we're encrypting.
```

Comment:
> by an encrypted value

nit: hashing with salt isn't encryption. Encryption means we can take the cipher text and apply a key + a function to go back to the original state. Here with hashing + salt, we're ensuring that the computed text is unique and hard to guess but you can't go back to the original state (unless you have access to KV 😄 )

I would say something along the lines of. 
`In order to safely store user IDs (the `sub` value from Auth0) in the cookie we set in the browser, we should always refer to them by a value that cannot be easily guessed by someone else.`